src/encoded/static/components/genome_browser.js

```diff
@@ -155,6 +159,27 @@ function mapGenome(inputAssembly) {
     return genome;
 }
 
+// Map the name of a sort parameter to a file object property and convert to form that can be compared
+// Ordering by replicate is like this: 'Rep 1,2' -> 'Rep 1,3,...' -> 'Rep 2,3,...' -> 'Rep 1' -> 'Rep 2' -> 'Rep N'
+// Multiplication by 1000 orders the replicates with a single replicate at the end
+const sortLookUp = (obj, param) => {
```

Comment:
Switch statement may be better as you have more then three conditions and each condition is just one statement